CHANGELOG.md

```diff
@@ -19,7 +19,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
-(None)
+### Added
+
+- Added webdriver tasks for selecting/deselecting Checkboxes/Radio buttons
```

Comment:
Nitpick, but please improve the comments.
* webdriver --> WebDriver
* name each new task name so folks know how to find them

Reply:
Sounds good, I'll update this once we've decided on the name(s) of these 'checkbox' tasks